test/boulder-config.json

```diff
@@ -2,7 +2,8 @@
   "syslog": {
     "network": "",
     "server": "",
-    "stdoutlevel": 6
+    "stdoutlevel": 6,
+    "sysloglevel": 7
```

Comment:
Let's set sysloglevel for this test config to 4. That's nice because it allows us to stop spamming up syslog on developer machines.

Also, you should add the same change to boulder-config-next.json.